pangraph/Tuura/Pangraph/Main.hs

```diff
@@ -0,0 +1,27 @@
+module Tuura.Pangraph.Main (main) where
```

Comment:
This should live in a separate folder, say `Fantasi` instead of `Tuura.Pangraph`.